File: Arrays/2/mostWater.js
const mostWater = (height) => {
  let maxArea = 0
  for(let p1=0;p1<height.length - 1;p1++) {
    for(let p2=p1+1;p2<height.length;p2++) {
      const calcArea = Math.min(height[p1], height[p2]) * (p2 - p1);
      if (calcArea > maxArea) {
        maxArea = calcArea;
      }
    }
  }
  return maxArea
};

console.log(mostWater(height = [7, 1, 2, 3, 9]));

File: Arrays/2/mostWater.py
def maxArea(height):
    area = 0
    for p1 in range(len(height)-1):
            for p2 in range(p1+1, len(height)):
                calcArea = min(height[p1], height[p2]) * (p2-p1)
                if calcArea > area:
                    area = calcArea
    return area

print(maxArea(height=[7,1,2,3,9]))